alljoyn/Makefile

```diff
@@ -4,12 +4,14 @@ PKG_NAME:=alljoyn
 PKG_BASE_VERSION:=14.12
 PKG_PATCHLEVEL:=a
 PKG_RELEASE:=1
-PKG_MD5SUM:=1435d8611a26bfde115a771ab7cd3d74
-PKG_VERSION:=$(PKG_BASE_VERSION)$(PKG_PATCHLEVEL)
-PKG_TARBALL_VERSION:=$(PKG_BASE_VERSION).00$(PKG_PATCHLEVEL)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/alljoyn/core-alljoyn.git
+PKG_SOURCE_VERSION:=RB$(PKG_BASE_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_TARBALL_VERSION).tar.gz
```

Comment:
you're using PKG_TARBALL_VERSION here when you removed it above (see line 9, original).